test/clj/webchange/test/assets/handler.clj

```diff
@@ -0,0 +1,23 @@
+(ns webchange.test.assets.handler
+  (:require [clojure.test :refer :all]
+            [webchange.test.fixtures.core :as f]
+            [webchange.assets.loader :as loader]
+            [webchange.assets.core :as core]
+            [webchange.db.core :refer [*db*] :as db]
+            ))
+
+(use-fixtures :once f/init)
+(use-fixtures :each f/clear-db-fixture f/with-default-school)
+
+(deftest could-create-asset-hash
+  (loader/execute ["calc-asset-hash" ] {:dataset-dir "test/clj/webchange/resources"})
+  (is  (f/is-created-asset-hash? "6576fcab3f033a1e11a1971ee2a7cf54"))
+  )
+
+(deftest could-upload-asset
```

Comment:
please add one more test to check if image size it returned and it is correct one - this way we will make sure that our endpoint response is correct.